cmake/RunInputFileTest.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh -e
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Positional arguments to this script:
+# - $1: executable name
+# - $2: path to input file
+# - $3: directory name
+# - $4: additional command-line arguments forwarded to the executable
+# - $5: expected exit code
+# - $6: "true" to check output files or "false" to skip the check
```

Comment:
The code below is correct but these docs are wrong. 6 is the additional args, 4 is expected exit code, and 5 is check output.